docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -521,6 +522,10 @@ This is equivalent to the Podman `--pull` option
 
 If enabled, makes the image read-only.
 
+### `ReadOnlyTmpfs=` (defaults to `yes`)
+
+If container is running in ReadOnly is set to yes, then mount a read-write tmpfs on /dev, /dev/shm, /run, /tmp, and /var/tmp.
```

Comment:
```suggestion
If container is running and ReadOnly is set to yes, then mount a read-write tmpfs on /dev, /dev/shm, /run, /tmp, and /var/tmp.
```

Should Quadlet error out if this option is set but `ReadOnlyTmpfs` is not (or turned off)?

Reply:
I've rephrased the sentence.

> Should Quadlet error out if this option is set but ReadOnlyTmpfs is not (or turned off)?

In general, Quadlet does not validate the values or their relationship unless needed (e.g. `NetworkSubnet`, `NetworkGateway` and `KeyNetworkIPRange`). Furthermore, in this specific case, Podman does not fail if `--read-only-tmpfs` is set while `--read-only` isn't. So, I don't think Quadlet should validate the values